### ISSUE TITLE
configurations for duplicating color+gray model

### ISSUE THREAD

**Issue by gongbudaizhe** [bertinetto/siamese-fc#27]
Hi,

As the paper states, for OTB benchmark, 25% color images are randomly converted to gray images. The pretrained model is named color+gray in the project homepage:
http://www.robots.ox.ac.uk/~luca/stuff/siam-fc_nets/2016-08-17_gray025.net.mat.

Currently, I am trying to replicate training this model using configurations like:

```matlab
    opts.augment.grayscale = 0.25
```

all other configurations remains untouched. But somehow, after training 50 epoches,  I only get AUC: 0.577 using 3 scales and AUC: 0.584 using 5 scales at the OTB cvpr13 test set.

However, using the pretrained model, I get AUC: 0.582 using 3 scales, AUC: 0.605 using 5 scales.

Is there anything I am missing?

**Comment by gongbudaizhe** [comment]
I have replicated color + gray with simple modifications:

1. instead of using xavierimproved initialization, I directly use xavier initialization.
2. instead of get random number of gray images for each batch, I use fixed ratio gray images, that is 2 gray images and 6 color images for each batch.

I now have AUC: 0.606194263434.

**Comment by bertinetto** [comment]
Ok great!
Thanks for pointing this out!
